apps/examples/lcd_test/example_lcd.c

```diff
@@ -360,6 +365,55 @@ static void test_fps(void)
 	}
 }
 
+static void touch_test(void)
+{
+	/* read first 10 events */
+	char c;
+	int ret;
+
+	int fd = open("/dev/input0", O_RDONLY);
+	printf("in touch test\n");
+	if (fd < 0) {
+		printf("Failed to open /dev/input0, errno : %d\n", get_errno());
+		return NULL;
+	}
+
+	struct pollfd fds[1];
+	fds[0].fd = fd;
+	fds[0].events = POLLIN;
+
+	struct touch_sample_s buf;
+	for (int i = 0; i < 100; i++) {
```

Comment:
What's the meaning of 100? After 100 touch, this task will be automatically finished?
If yes, you already added task_delete at end of lcd test main function. So, how about changing this to infinite loop?